Features.md

```diff
@@ -14,27 +14,308 @@
+
+CISA recommends limiting 
+the document to three commonly used formats to facilitate widespread adoption.
+
+
+***Remediation***
+- Re-create the document in CycloneDX, SPDX, or SWID.
+---
+#### Specification Version 
+
+This check determines whether the given SBOM is in the specification version that can support fields necessary for typical SBOM operations. 
+The current check tests for 
+- CycloneDX Versions: 1.0, 1.1, 1.2, 1.3, 1.4
+- SPDX Versions: 2.1, 2.2, 2.3
+
+While the earlier versions of specifications may exist, a document in an earlier version will not be able to carry all of the required fields.
+
+***Remediation***
+- Re-create the document in one of the versions listed above.
+---
+#### Specification File Format 
+
+This check determines whether the given SBOM can be easily consumed by testing for the most common file formats associated with the specification.
+- CycloneDX: XML, JSON
+- SPDX: JSON, YAML, RDF, tag/value
+
+Building and sharing SBOM in the most commonly used file format enables the use of SBOM in various conditions.
+
+***Remediation steps***
+- Re-create the document in one of the file formats listed above.
+
+---
+#### Specification Syntax 
+
+This check determines whether the given SBOM meets all the requirements of the underlying specification and file format to be parsed.
+
+A syntactic error in the SBOM will prevent it from being usable.
+
+***Remediation***
+- Check the SBOM generator tool's known issues and get the most recent version of the tool.
+- Check options/setup of the environment variables required to use the tool.
+- Build SBOM with a different tool.
+
+---
+#### SBOM Specification Syntax 
+
+This check determines whether the given SBOM meets all the requirements of the underlying specification and file format to be parsed.
+
+A syntactic error in the SBOM will prevent it from being usable.
+
+***Remediation***
+- Check the SBOM generator tool's known issues and get the most recent version of the tool.
+- Check options/setup of the environment variables required to use the tool.
+- Build SBOM with a different tool.
+
+---
+### Category: NTIA-Minimum-Elements
+---
+#### Component Name 
+
+This check determines whether each component in the SBOM includes a name.
+
+Components must have a name to be used meaningfully to assess compliance or security risk.
+
+**Remediation**
+
+Identify the component with a missing name and check its product page to get its name.
+- CycloneDX field: [components:name](https://cyclonedx.org/docs/1.4/json/#components_items_name)
+- SPDX field: [PackageName](https://spdx.github.io/spdx-spec/v2.3/package-information/#71-package-name-field)
+
+---
+#### Supplier Name 
+
+This check determines whether each component in the SBOM includes a supplier name.
```

Comment:
We should ack, there is ongoing dicusssions in the open-source ecosystem, about open-source maintainers being classified as suppliers ?

Reply:
Added